params/network.go

```diff
@@ -16,6 +18,9 @@ const (
 // Network is a type definition for DA network run by Celestia Node.
 type Network string
 
+// BootstrapPeers is a type definition for nodes that will be used as bootstrappers
+type BootstrapPeers []*peer.AddrInfo
```

Comment:
We need to be consistent with namings. Our func in the pkg is called BootstrappersFor while the type is BootstrapPeer.

Reply:
Also, now I think we should keep peer.AddrInfo as a value(not pointer) to be consistent with DHT and they libp2p uses them